src/main/kotlin/com/ekino/oss/recaptcha/service/ReCaptchaValidationService.kt

```diff
@@ -0,0 +1,37 @@
+package com.ekino.oss.recaptcha.service
+
+import com.ekino.oss.recaptcha.client.ReCaptchaClient
+import com.ekino.oss.recaptcha.config.ReCaptchaProperties
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import java.io.IOException
+
+private val logger = KotlinLogging.logger {}
+
+@Service
```

Comment:
this isn't needed since the service is imported with `@Import`. It's possible that the service will be created even if the configuration is disabled

Reply:
Indeed, I removed it